src/common/apple/subscription.ts

```diff
@@ -164,7 +165,11 @@ const renewalInfoToSubscriptionFlags = (
 
   if (isNullOrUndefined(cycle)) {
     logger.error(
-      { data, provider: SubscriptionProvider.AppleStoreKit },
+      {
+        data,
+        provider: SubscriptionProvider.AppleStoreKit,
+        processor: SubscriptionProcessor.Plus,
```

Comment:
processor means something else in UserTransaction, maybe call it `productType`, or something, just so it not overlaps

Reply:
Yeah, will do

Reply:
I renamed the `SubscriptionProcessor` enum to `PurchaseType`.